Templates/GitHubActionsPipeline/.github/workflows/pipelineController.yml

```diff
@@ -0,0 +1,86 @@
+name: Pipeline_Controller
+on: 
+    workflow_dispatch:
+        inputs:
+            pipelineType:
+                description: Pipeline Type
+                default: build
```

Comment:
Let the developer choose from a list of options for the pipeline type like on the release type.

The options are:
build, preview, release; with build as the default